pkgs/development/compilers/inform6/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "inform6";
+  version = "6.34-6.12.2";
+
+  src = fetchurl  {
+    url = "https://ifarchive.org/if-archive/infocom/compilers/inform6/source/inform-${version}.tar.gz";
+    sha256 = "c149f143f2c29a4cb071e578afef8097647cc9e823f7fcfab518ac321d9d259f";
+  };
+
+  buildInputs = [ perl ];
+
+  buildPhase = ''
+    make PREFIX=$out
+  '';
```

Comment:
```suggestion
  buildFlags = [ "PREFIX=${placeholder "out"}" ];
```

Reply:
Thanks for the tip; I amended my commit to use `buildFlags`